src/data/Board.js

```diff
@@ -1,12 +1,76 @@
 export const BOARD = [
```

Comment:
Can you import the cat image from public/assets and then pass that into the image prop for all the Board.